pawprint/sessions.py

```diff
@@ -0,0 +1,82 @@
+import pandas as pd
+
+
+class UserCurrentSession:
+    """One user's current session. This stores the start of the session,
+    the number of events in the session, and the name of the event
+    """
+
+    def __init__(self, user_id, start_time, events, last_time=None):
+        self.user_id = user_id
+        if last_time:
+            self.last_timestamp = last_time
+        else:
+            self.last_timestamp = start_time
+        self.first_timestamp = start_time
+
+        self.events_in_session = events + ","
+        # if isinstance(events, list):
+        #     self.events_in_session = events
+        # else:
+        #     self.events_in_session = [events]
+
+    def log(self, event_timestamp, events):
+        # if new event within 30 min of last event for user,
+        # update session end timestamp with this time and
+        # add one to event count
+        if (event_timestamp - self.last_timestamp).seconds < 30 * 60:
```

Comment:
We have session duration as a user-defined variable in `stats.sessions()`. Do we want this to be a kwarg with default value of 30 ?